SU-HSDO/suhumsci
# Arch 296 space below footer shadow

# READY FOR REVIEW

# Summary
- Resolving odd footer box shadow issues at lower breakpoints.

# Steps to Test
1. Check out the footer appearance at all breakpoints
1. See what you think about the inner shadow as opposed to outer (see notes in ticket)

# Associated Issues and/or People
- [JIRA ticket](https://stanfordits.atlassian.net/browse/ARCH-296)
